pkgs/tools/typesetting/stapler/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, buildPythonApplication, fetchFromGitHub, pypdf2, more-itertools }:
+
+buildPythonApplication rec {
+  pname = "stapler";
+  version = "2016-04-24";
+  name = "${pname}-${version}";
```

Comment:
Since this is an unreleased version the name should be `"${pname}-unstable-${version}"`.